server/src/interfaces/job.interface.ts

```diff
@@ -142,12 +148,20 @@ export interface IEmailJob {
   subject: string;
   html: string;
   text: string;
+  attachments?: Attachment[];
 }
 
 export interface INotifySignupJob extends IEntityJob {
   tempPassword?: string;
 }
 
+export interface INotifyAlbumInviteJob {
+  album: AlbumEntity;
+  guestUser: UserEntity;
+}
```

Comment:
I would probably just extend `IEntityJob` and pass `id: album.id` instead of the whole album entity.